README.md

```diff
@@ -18,10 +18,198 @@ Or install it yourself as:
 
     $ gem install litmus_paper
 
+## Overview
+
+Litmus Paper reports health for each service on a node on a 0-100 scale. Health is computed by aggregating the values returned from running various subchecks.
+
```

Comment:
Add: `This information can be queried on the command line, or returned via an HTTP API for consumption by other services (like load balancers, or monitoring systems).`